packages/core/src/tools/heavyCustomerDataWarning.ts

```diff
@@ -0,0 +1,21 @@
+import { display } from './display'
+import { ONE_KIBI_BYTE } from './utils'
+
+export const CUSTOMER_DATA_BYTES_LIMIT = 3 * ONE_KIBI_BYTE
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you add a comment with a bit of reasoning on the value?